frontend/package.json

```diff
@@ -10,24 +10,24 @@
 		"lint": "eslint src mock-db adaptors",
 		"build": "vite build && vite build -c adaptors/cloudflare-pages/vite.config.ts",
 		"dev": "vite --mode ssr",
-		"watch": "concurrently \"vite build -w\" \"vite build -w -c adaptors/cloudflare-pages/vite.config.ts\""
+		"watch": "nodemon -w ./src --ext tsx,ts --exec npm run build"
```

Comment:
This is slower but works better then the current watch, with the current watch I very very often get qwik to break and not handle well user interaction, with this new version we do always a fresh build so everything keeps working (although the continuous fresh re-build is slower of course)

@jculvey have you also noticed the current `watch` breaking qwik?

Reply:
yeah, it's incredibly frustrating. excited if this fixes it

Reply:
yeah it does 😃 
(but as I mentioned this is slower of course 😢)